.github/workflows/terraform-ci.yml

```diff
@@ -99,7 +99,7 @@ jobs:
           - acceptance-1.15-EC2
         include:
           - name: acceptance-1.15-FARGATE-HCP
-            consul-version: '1.15.10'
+            consul-version: '1.15.9'
```

Comment:
Is this intended to decrease the version?

Reply:
Yes. `1.15.10` isn't yet available in HCP